src/medCore/database/medDataManager.cpp

```diff
@@ -249,6 +249,8 @@ void medDataManager::launchExporter(medDatabaseExporter* exporter, const QString
     connect(exporter, SIGNAL(progressed(int)), message, SLOT(setProgress(int)));
     connect(exporter, SIGNAL(success(QObject *)), message, SLOT(success()));
     connect(exporter, SIGNAL(failure(QObject *)), message, SLOT(failure()));
+    connect(exporter, SIGNAL(success(QObject *)), this, SIGNAL(dataExported()));
+    connect(exporter, SIGNAL(failure(QObject *)), this, SIGNAL(dataExported()));
```

Comment:
Since the signal is emitted in case of failure and success, I'd rather call the signal ```exportFinished```.